classes/class-wc-connect-nux.php

```diff
@@ -94,8 +94,10 @@ public function show_pointers( $hook ) {
 			}
 
 			wp_enqueue_style( 'wp-pointer' );
-			wp_enqueue_script( 'wc_services_admin_pointers', plugins_url( 'assets/js/admin-pointers.js', dirname(__FILE__) ), array( 'wp-pointer', 'jquery' ) );
+			$wc_connect_base_url = defined( 'WOOCOMMERCE_CONNECT_DEV_SERVER_URL' ) ? WOOCOMMERCE_CONNECT_DEV_SERVER_URL : plugins_url( 'dist/', __FILE__ );
+			wp_register_script( 'wc_services_admin_pointers', $wc_connect_base_url . 'woocommerce-services-admin-pointers.js', array( 'wp-pointer', 'jquery' ), false, true );
```

Comment:
I agree with you, @DanReyLop - and I made this change in https://github.com/Automattic/woocommerce-services/pull/843/commits/1fca8276d3731d95c6fbd0752bf6b2a9a216230c. We may need to update `class-wc-connect-nux.php#add_shared_notice_styles()` later as well, @marcinbot.